src/components/about/xoomInOutView.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
```

Comment:
Can you export this as a separate component, so we can use this component independently in future

Reply:
@AjayDoddapaneni Yes it's already exported as separate component.